### DISCUSSION TITLE
Something wrong with column 'timestamps' in train_valid_sequences.csv

### DISCUSSION THREAD

**Discussion by su-sin-yu** [pykt-team/pykt-toolkit#67] [Q&A] [answered]
When I try to transform the values in 'timestamps' to the date, found that there are some strange values, which including -1.
Just like the screenshot below : 
<img width="665" alt="image" src="https://user-images.githubusercontent.com/79685867/202015784-4726e95b-3ec1-4837-abd8-81ab468cbbe1.png">

Just want to know is this situation normal or there might be the somewhere wrong with the data. Thanks.

**Comment by Liu-lqq** [comment, marked as answer]
Thank you for your attention. 
This is because we processed the data into a format that the model can directly use, when the sequence length is less than 200, we will pad -1 to it to make the length of all sequences equal 200.
If you want to get the original sequence, you can see the file "train_valid.csv", but this file can't be used directly by the models.
You can find the processed details in the file "pykt/preprocess/split_datasets.py"
